src/main/java/nl/ordina/jobcrawler/controller/VacancyScraper.java

```diff
@@ -1,19 +1,61 @@
 package nl.ordina.jobcrawler.controller;
 
+import lombok.extern.slf4j.Slf4j;
 import nl.ordina.jobcrawler.model.Vacancy;
 import nl.ordina.jobcrawler.model.VacancyURLs;
+import nl.ordina.jobcrawler.service.ConnectionDocumentService;
 import org.jsoup.nodes.Document;
+import org.springframework.beans.factory.annotation.Autowired;
 
 import java.io.IOException;
+import java.util.ArrayList;
 import java.util.List;
 
 // All vacancy scrapers have the following functions in common
 
+@Slf4j
 abstract class VacancyScraper {
 
-    protected static final String SEARCH_TERM = "java";
+    protected String BROKER = "";
+    protected String SEARCH_URL = "";
+    protected final String SEARCH_TERM = "java";
```

Comment:
I'd rather see abstract getters.

for example:
```
abstract String getBroker();
```

Reply:
Earlier this week we decided to add the SEARCH_URL and the BROKER to the constructor of the abstract class. That way we can make the variables private final and they need to be added in the constructor of the scrapers.